sunpy/map/maputils.py

```diff
@@ -98,8 +98,9 @@ def contains_full_disk(smap):
 
     Notes
     -----
-    * The function assumes that the input map is rectangular.
-    * In the case of coronagraph images the disk itself need not be observed.
+    * The disk itself need not be imaged.  For example, in coronagraph images such as those
```

Comment:
The `*` here make this bullet points, I think you just want to drop them.

Reply:
Done